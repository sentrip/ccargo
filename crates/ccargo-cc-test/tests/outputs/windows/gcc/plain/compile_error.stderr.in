main.c: In function 'main':
main.c:4:13: error: void value not ignored as it ought to be
     int x = func();
             ^~~~
main.c:5:13: error: void value not ignored as it ought to be
     int y = func();
             ^~~~
