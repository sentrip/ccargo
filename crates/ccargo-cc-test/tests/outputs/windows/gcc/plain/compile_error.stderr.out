error: void value not ignored as it ought to be
   --> main.c:4:13 in function `'main'`
     int x = func();
             ^~~~
error: void value not ignored as it ought to be
   --> main.c:5:13 in function `'main'`
     int y = func();
             ^~~~
